.jshintrc

```diff
@@ -4,6 +4,7 @@
   "maxparams": 4,
   "maxcomplexity": 13,
   "globals": {
+    "before": true,
```

Comment:
Although I don’t use this anymore, I figure it might be valuable for the future :smile: